src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,284 @@
+update the `pubspec.lock` file.
+You can then run `pub outdated` to confirm that
+you've made all necessary changes.
+In this example, the `path` and `meta` packages are still out-of-date,
+due to reasons that this package doesn't control:
+
+```terminal
+$ pub upgrade
+...
+$ pub outdated
+Dependencies  Current  Upgradable  Resolvable  Latest  
+path          1.6.2    1.6.2       1.6.2       1.7.0   
+
+dev_dependencies: all up-to-date
+
+transitive dependencies
+meta          1.1.6    1.1.6       1.1.6       1.1.8   
+
+transitive dev_dependencies: all up-to-date
+
+Dependencies are all on the latest resolvable versions.
+Newer versions, while available, are not mutually compatible.
+$
+```
+
+To see why these packages are out-of-date, you can run `pub deps`
+and look for dependencies on these packages:
+
+```terminal
+$ pub deps -s list
+...
+dependencies:
+...
+- terminal_tools 0.1.0
+  - path 1.6.2
+  - meta 1.1.6
+...
+$ 
+```
+
+As the preceding output shows,
+this package depends on the `terminal_tools` package,
+which depends on old versions of `path` and `meta`.
+Once the `terminal_tools` package is updated,
+it should be possible to update this package.
+
+{{site.alert.important}}
+  Test your code to verify that it still works as expected
+  after updating the packages.
+{{site.alert.end}}
+
+
+## Output columns
+
+The output of `pub outdated` has four columns of version information
+for each out-of-date dependency.
+The following example
+(which has the same text as the screenshot in the [Example section](#example))
+shows the four columns:
+Current, Upgradable, Resolvable, and Latest.
+
+```terminal
+$ pub outdated
+Dependencies  Current    Upgradable  Resolvable  Latest  
+args          1.4.4      1.6.0       1.6.0       1.6.0   
+http          0.11.3+17  0.11.3+17   0.12.1      0.12.1  
+path          1.6.2      1.6.2       1.6.2       1.7.0   
+
+dev_dependencies: all up-to-date
+
+transitive dependencies
+meta          1.1.6      1.1.6       1.1.6       1.1.8   
+
+transitive dev_dependencies: all up-to-date
+
+1 upgradable dependency is locked (in pubspec.lock) to an older version.
+To update it, use `pub upgrade`.
+
+1 dependency is constrained to a version that is older than a resolvable version.
+To update it, edit pubspec.yaml.
+$
+```
+
+Current
+: The version used in your package, as recorded in `pubspec.lock`.
+  If the package isn't in `pubspec.lock`,
+  the value is `-`.
+
+Upgradable
+: The latest version allowed by your `pubspec.yaml` file.
+  This is the version that `pub upgrade` resolves to.
+  The value is `-` if the value in the **Current** column is `-`.
+
+Resolvable
+: The latest version that can be resolved,
+  when combined with all other dependencies.
+  This version corresponds to what `pub upgrade` would give you
+  if all version constraints in `pubspec.yaml` were `any`.
+  A value of `-` means that the package won't be needed.
+
+{{site.alert.important}}
```

Comment:
I personally find the big yellow box here in the middle of the column explanations a bit confusing (I thought for a sec. that Latest wasn't described). Can we consider either 1) moving this below the description of Latest, or 2) rephrasing the explanation of Resolvable to not use `any` (e.g.: "The latest version that can be resolved, when combined with all other dependencies. This version corresponds to what pub upgrade would give you if all version constraints in pubspec.yaml were unbounded.")